src/components/TeamSection/teamMembers.json

```diff
@@ -131,6 +131,20 @@
       "linkedin": "https://www.linkedin.com/in/ammerhamza/"
     }
   ],
+  "alumniTeam": [
```

Comment:
Probably just wanted to show the UI, but just a reminder to change these to their correct contact :)

Reply:
Yes I had made note of that in the description of the PR

Reply:
The headshots for Alumni were added just today so I wasn't able to update it before, but thanks for reminding!